dbt_project.yml

```diff
@@ -0,0 +1,50 @@
+
+name: 'fractribution'
+version: '1.0.0'
+config-version: 2
+
+# This setting configures which "profile" dbt uses for this project.
+profile: 'fractribution'
+
+
+# These configurations specify where dbt should look for different types of files.
+# The `model-paths` config, for example, states that models in this project can be
+# found in the "models/" directory. You probably won't need to change these!
+model-paths: ["models"]
+analysis-paths: ["analyses"]
+test-paths: ["tests"]
+seed-paths: ["seeds"]
+macro-paths: ["macros"]
+snapshot-paths: ["snapshots"]
+
+target-path: "target"  # directory which will store compiled SQL files
+clean-targets:         # directories to be removed by `dbt clean`
+  - "target"
+  - "dbt_packages"
+
+vars:
+  conversion_window_start_date: '2022-06-03' # conversion window start date
+  conversion_window_end_date: '2022-08-01' # conversion window end date
```

Comment:
Thanks Agnes, this is a good point. Mike suggested current_date()-1 may be better since current_date won't have all the data yet. I will make the changes for this.